packages/postgres-database/src/simcore_postgres_database/utils_payments_autorecharge.py

```diff
@@ -0,0 +1,94 @@
+import sqlalchemy as sa
+from simcore_postgres_database.models.payments_autorecharge import payments_autorecharge
+from simcore_postgres_database.models.payments_methods import (
+    InitPromptAckFlowState,
+    payments_methods,
+)
+from sqlalchemy.dialects.postgresql import insert as pg_insert
+
+
+class AutoRechargeStmt:
```

Comment:
Q: maybe plural reads better?
```suggestion
class AutoRechargeStmts:
```